Code/Unsupervised_reconstruction/data/reconstruction_datasets.py

```diff
@@ -23,7 +25,10 @@ def __training__collate_fn(batch:list[tuple[str, dict[ModernLanguages, str], __C
 
     return (firstElement, secondElement, lastElement)
 
-class __TrainingDataset(Dataset):
+class __TrainingDataset(Dataset[tuple[str,
```

Comment:
Is it not too lengthy for a type? Would a local alias before the definition not be justified here?

Reply:
the next commit has fixed that 👍